sources/core/tests/fast_serializer_tests.cpp

```diff
@@ -660,6 +660,10 @@ TEST(FastSerializeDeserialize, VariousValues)
         serializer << QString("One two three");
         serializer << QString("Ехал грека через реку");
         serializer << 4294967294u;
+        serializer << std::bitset<4>(0);
```

Comment:
Alongside with the code of serialization functions the tests should be moved to appropriate file: `saveable_operators_tests.cpp`.
Also if it is `std::bitset<4>` then there are should be 16 possible values, not 4, so it may be wise to check them all.